src/app/services/fake-goal-management-service.ts

```diff
@@ -0,0 +1,54 @@
+import {Goal} from '../model/goal';
+
+export class FakeGoalManagementService {
+  goal: Goal;
+  goals = new Map<string, Goal>();
+  userGoals = new Map<string, Goal[]>();
+
+  constructor() {}
+
+  setGoal(
+      name: string, startDate: Date, endDate: Date, workload: number,
+      avgWorkload: number) {
+    let id = this.goals.size;
+    let diffDays =
+        (endDate.valueOf() - startDate.valueOf()) / (1000 * 3600 * 24);
+    let dailyProgress = new Array<number>(diffDays);
+    let groups = new Array<number>();
```

Comment:
same here. Prefer:
let groups: number[] = [];

Reply:
ok got it